src/invidious/videos.cr

```diff
@@ -570,6 +570,10 @@ struct Video
     info["authorThumbnail"]?.try &.as_s || ""
   end
 
+  def author_verified : Bool
+    info["authorVerified"].as_bool
```

Comment:
please use `.try &.as_bool || false`, as the video parsing should fail as nicely as possible.
We don't want the users to be sent to the "you found a bug" page for just a missing field ^^